src/components/menu/menu.spec.ts

```diff
@@ -376,6 +382,32 @@ describe('MenuComponent', () => {
 
         expect(component.item1Activated).toHaveBeenCalledTimes(1);
     });
+
+    it('should not emit a click event when disabled is true.', async () => {
+        component.disabled = true;
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        // open menu
+        component.trigger.openMenu();
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        // get the child submenu item
+        const menuItem = document.querySelector<HTMLButtonElement>(
+            '#menu-item-disabled'
+        );
+
+        expect(menuItem).toBeTruthy();
+        spyOn(component, 'onButtonClick');
```

Comment:
We have an `activate` output as well, we should assert that that does not get called when the menu item is disabled. You could just use the first menu item in that case because it already has an activate handler.

Reply:
Would it be better to have them as two separate tests or just include them in the one test?

Reply:
It's probably fine to check both in the same test, it's just one extra check.